### ISSUE TITLE
Error when installing Docksal on a machine without SSH keys

### ISSUE THREAD

**Issue by lmakarov** [docksal/docksal#365]
This error shows up anytime Docksal tries to load SSH keys into the ssh-agent, but the `~/.ssh` folder is not there (the host has no SSH keys).

`GetFileAttributesEx C:\Users\admin\.babun\cygwin\home\admin\.ssh: The system cannot find the file specified.`

During first-time install:

```
...
Resetting network settings...
Resetting Docksal services...
 * proxy
 * dns and resolver for .docksal domain
Enabling automatic *.docksal DNS resolver...
Clearing DNS cache...
 * ssh-agent
GetFileAttributesEx C:\Users\admin\.babun\cygwin\home\admin\.ssh: The system cannot find the file sp                         ecified.
Configuring SMB shares...
Enter your Windows account password:
Adding docker SMB share...
Importing ssh keys...
GetFileAttributesEx C:\Users\admin\.babun\cygwin\home\admin\.ssh: The system cannot find the file specified.
```

When running `fin reset ssh-agent`:

```
Resetting Docksal ssh-agent service...
GetFileAttributesEx C:\Users\admin\.babun\cygwin\home\admin\.ssh: The system cannot find the file specified.
```

**Comment by achekulaev** [comment]
I guess we need to append `mkdir -p $HOME/.ssh` to `install_tools_windows`

**Comment by lmakarov** [comment]
Fixed in b981b089ef5a8012be81b04ac9e3ae03ca48ae3a
